src/CommandLineArgumentsParser/CommandLineArgumentsParser.csproj

```diff
@@ -30,30 +30,9 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    </ItemGroup>
-
-    <ItemGroup Condition=" '$(TargetFramework)' == 'net20' ">
-        <PackageReference Include="LinqBridge" Version="1.3.0" />
-    </ItemGroup>
-
-    <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-        <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
-    </PropertyGroup>
-
-    <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
-        <PackageReference Include="Microsoft.CSharp" Version="4.3.0" />
-        <PackageReference Include="System.Collections" Version="4.3.0" />
-        <PackageReference Include="System.Console" Version="4.3.0" />
-        <PackageReference Include="System.Globalization" Version="4.3.0" />
-        <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
-        <PackageReference Include="System.Linq" Version="4.3.0" />
-        <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
-        <PackageReference Include="System.Reflection.TypeExtensions" Version="4.3.0" />
-        <PackageReference Include="System.Resources.ResourceManager" Version="4.3.0" />
-        <PackageReference Include="System.Runtime.Extensions" Version="4.3.0" />
-        <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
-        <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
+        <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
+        <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
```

Comment:
Instead of depending on Microsoft.Extensions.Logging

Just create a simple interface, like here:
https://github.com/WireMock-Net/WireMock.Net/blob/master/src/WireMock.Net.Abstractions/Logging/IWireMockLogger.cs

The name for the interface could just be `ICommandLineParserLogger`.

And implement a default Console Logger (e.g. https://github.com/WireMock-Net/WireMock.Net/blob/master/src/WireMock.Net/Logging/WireMockConsoleLogger.cs) which is used when nothing is provided to the commandline parser (so old behaviour stays the same)

Reply:
I meant to use only Microsoft.Extensions.Logging.Abstractions. I will remove the Microsoft.Extensions.Logging dependency. Abstractions package will allow consumer to use any logging packages (Serilog, NLog etc). There is no need to create new interface.

Also console logger was just a sample to use for test. Consumers can be writing the log to any sink (file, eventlog etc) depending on their need. We don't need to do custom implementation.